randomisation/models.py

```diff
@@ -0,0 +1,55 @@
+from django.db import models
+from django.db.models import JSONField
+from django.utils.text import slugify
+
+
+class Strata(models.Model):
```

Comment:
I changed these models a bit now. Strategy isn't a parent of Strata anymore. You can now setup stratas and use them in multiple strategies.